.github/workflows/repotests.yml

```diff
@@ -89,6 +89,10 @@ jobs:
         with:
           ruby-version: '3.4'
           bundler-cache: true
+      - name: setup-cocoapods # Not using an action, as it is not implemented for windows!
+        run: |
+          gem install cocoapods -v 1.16.2 --no-document
```

Comment:
Can we add this to the various dockerfiles including Ruby ones?

Reply:
Sure, I don't see why not. In case you are already installing Ruby Gems (eg with Bundler & Gemfile), we could simply add it there. Or just call it like here.